hexa/core/analytics.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+from uuid import uuid4
+
+from django.http import HttpRequest
+from mixpanel import BufferedConsumer, Mixpanel, MixpanelException
+from sentry_sdk import capture_exception
+from ua_parser import user_agent_parser
+
+from config.settings.base import MIX_PANEL_BUFFER_SIZE, MIX_PANEL_TOKEN
```

Comment:
Nope.

`from django.conf import settings`

Reply:
It's the auto-import, sorry!